Update node-ingest to support latest version (#1314)
MicrosoftDocs/dataexplorer-docs: data-explorer/node-ingest-data.md

const kustoDatabase  = "Weather";
```

> Now construct the connection string. This example uses device authentication to access the cluster (Make sure to look at the console output to authenticate). You can also use Azure Active Directory application certificate, application key, and user and password.

Reviewer: ```suggestion
Now construct the connection string. This example uses device authentication to access the cluster. Check the console output to complete the authentication. You can also use an Azure Active Directory application certificate, application key, and user and password.
```
